types/custom_trusted_oid_mapping.pp

```diff
@@ -0,0 +1,4 @@
+type Puppet::Custom_trusted_oid_mapping = Hash[String, Struct[{
```

Comment:
We have a bug in our installer that it currently doesn't handle multi line types because it does incorrect manual parsing. We should be fixing it but in the short term I wonder if we should put it on a single line.

Reply:
Sure!